build_tools/travis/install.sh

```diff
@@ -1,107 +1,71 @@
 if [[ "$USE_PROJECT" == "0" ]]; then
-  cd OpenMEEG
+    cd OpenMEEG
 fi
 
 mkdir build
 cd build
 
 # XXX : BUILD_SHARED should be used to set global defaults
 
-function install_matio {  # Install MATIO
-  wget https://github.com/openmeeg/matio-openmeeg/archive/master.zip
-  unzip master.zip
-  cd matio-openmeeg-master
-  mkdir build
-  cd build
-  cmake -DMAT73:BOOL=ON ..
-  make
-  sudo make install
-  cd ../../
-}
-
-if [[ $USE_PYTHON == "1" ]]; then
-  ENABLE_PYTHON=ON
-else
-  ENABLE_PYTHON=OFF
-fi
-
-if [[ $USE_VTK == "1" ]]; then
-  USE_VTK=ON
-else
-  USE_VTK=OFF
-fi
-
 if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
-  if [[ "$USE_PROJECT" == "0" ]]; then
-      # Build OpenMEEG and use openblas from homebrew
-      # or use Atlas which maps to vecLib or Accelerate frameworks
-      cmake \
-      -DBUILD_SHARED:BOOL=ON \
-      -DBUILD_DOCUMENTATION:BOOL=OFF \
-      -DBUILD_TESTING:BOOL=ON \
-      -DENABLE_PYTHON:BOOL=${ENABLE_PYTHON} \
-      -DENABLE_PACKAGING:BOOL=ON \
-      -DUSE_VTK:BOOL=${USE_VTK} \
-      -DBLASLAPACK_IMPLEMENTATION:BOOL=${BLASLAPACK_IMPLEMENTATION} \
-      -DCMAKE_SKIP_RPATH:BOOL=OFF \
-      ..
-  else
-      cmake \
-      -DBUILD_DOCUMENTATION:BOOL=OFF \
-      -DBUILD_TESTING:BOOL=ON \
-      -DENABLE_PYTHON:BOOL=${ENABLE_PYTHON} \
-      -DENABLE_PACKAGING:BOOL=ON \
-      -DUSE_VTK:BOOL=${USE_VTK} \
-      -DBLASLAPACK_IMPLEMENTATION:BOOL=${BLASLAPACK_IMPLEMENTATION} \
-      -DUSE_SYSTEM_matio:BOOL=OFF \
-      -DUSE_SYSTEM_hdf5:BOOL=OFF \
-      -DCMAKE_SKIP_RPATH:BOOL=OFF \
-      ..
-  fi
+    if [[ "$USE_PROJECT" == "0" ]]; then
+        # Build OpenMEEG and use OpenBLAS from homebrew
+        # or use Atlas which maps to vecLib or Accelerate frameworks
+        cmake \
+            -DBUILD_SHARED:BOOL=ON \
+            -DBUILD_DOCUMENTATION:BOOL=${BUILD_DOCUMENTATION} \
+            -DBUILD_TESTING:BOOL=ON \
+            -DENABLE_PYTHON:BOOL=${USE_PYTHON} \
+            -DENABLE_PACKAGING:BOOL=ON \
+            -DUSE_VTK:BOOL=${USE_VTK} \
+            -DUSE_CGAL:BOOL=${USE_CGAL} \
+            -DBLASLAPACK_IMPLEMENTATION:BOOL=${BLASLAPACK_IMPLEMENTATION} \
+            -DCMAKE_SKIP_RPATH:BOOL=OFF \
+            ..
+    else
+        cmake \
+            -DBUILD_DOCUMENTATION:BOOL=${BUILD_DOCUMENTATION} \
+            -DBUILD_TESTING:BOOL=ON \
+            -DENABLE_PYTHON:BOOL=${USE_PYTHON} \
+            -DENABLE_PACKAGING:BOOL=ON \
+            -DUSE_VTK:BOOL=${USE_VTK} \
+            -DUSE_CGAL:BOOL=${USE_CGAL} \
+            -DBLASLAPACK_IMPLEMENTATION:BOOL=${BLASLAPACK_IMPLEMENTATION} \
+            -DUSE_SYSTEM_matio:BOOL=${USE_SYSTEM} \
+            -DUSE_SYSTEM_hdf5:BOOL=${USE_SYSTEM} \
+            -DCMAKE_SKIP_RPATH:BOOL=OFF \
+            ..
+    fi
```

Comment:
Can we group the options to factorize the common ones across the two branches ...